components/ui/Copilot.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import {StyleProp, View, ViewStyle} from 'react-native';
+import {CopilotStep, walkthroughable} from 'react-native-copilot';
+
+export const Copilot: React.FC<CopilotProps> = (props: CopilotProps) => {
+  const CopilotView = walkthroughable(View);
+
+  return (
+    <CopilotStep
```

Comment:
TestIDs for Copilot? Can we identify the copilot overlay for different flows as of now in automation?

Reply:
Share card description , title, buttons are all having testIds so this can be ignored